packages/builder/src/builtin-tasks/clean.ts

```diff
@@ -9,6 +9,9 @@ export default function () {
     TASK_CLEAN,
     "Clears the cache and deletes all artifacts",
     async (_, { config }) => {
+      if (!config.paths) {
```

Comment:
Unfortunately these checks either have to be performed for each task or there is a need for one more type of a task. Current tasks were designed to be run only with a present config. This allows to run them without the config and a project.